### PR TITLE
more helpful instructions

### PR DESCRIPTION
**Pull request by botany-lead**
more helpful instructions

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 README
-plz
+
```
Comment (botany-demo) [inline readme.md:2]: clarity maximized 👍

Comment (botany-demo) [reply readme.md:2]: this is perfect

Comment (botany-demo) [reply readme.md:2]: hey @botany-lead how are you?

Comment (botany-demo) [reply readme.md:2]: hey @botany-lead can you check this out?
cc @botany-demo @botany-dummy

Comment (botany-demo) [reply readme.md:2]: this is really helpful

Comment (botany-lead) [reply readme.md:2]: i couldn't agree more

**Pull request merged**